src/main/java/com/commercetools/project/sync/product/ProductSyncer.java

```diff
@@ -328,10 +167,13 @@ protected ProductQuery getQuery() {
   static List<UpdateAction<Product>> appendPublishIfPublished(
       @Nonnull final List<UpdateAction<Product>> updateActions,
       @Nonnull final ProductDraft srcProductDraft,
-      @Nonnull final Product targetProduct) {
+      @Nonnull final ProductProjection targetProduct) {
 
+    // TODO: (ahmetoz) adapt queries.
```

Comment:
Those parts does not look ready for review 🤔  ?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/268/commits/4c2b27c91de43e712412a8bb6505c9b4ca57e28b
It was because of some branch conflicts, I had missed to push this change. Here it is now.
To explain, about this change, Yes, you are right. It is already handled in the library to append the update action for the product which is published.